pages/class/[id]/edit.vue

```diff
@@ -1,24 +1,45 @@
 <script setup lang="ts">
+// const { $useToast } = useNuxtApp()
+const toast = useToast();
+
+const show = () => {
+    toast.add({
+        severity: "secondary",
+        summary: "Url is copied to clipboard",
+        // detail: "Message Content",
```

Comment:
Are these code comments necessary? If not please remove it.

Reply:
ครับท่าน